modules/api/SshHelper.py

```diff
@@ -0,0 +1,52 @@
+import paramiko
+import logging
+import socket
+
+logger = logging.getLogger('esxi')
+
+# paramiko spams console with banner timeout exception but
+# exceptions should be handled here.
+logging.getLogger('paramiko.transport').setLevel(logging.CRITICAL)
+
+
+class SshHelper:
+
+    @staticmethod
+    def execute_command(address: str, user: str, password: str, command: str) -> str:
+        """
+        Runs a ssh command and returns str_out.
+
+        :param address: The address to the remote machine
+        :param user:  The ssh user
+        :param password:  The ssh password
+        :param command: The command to run
+        :return: Answer from host or None
+        """
+
+        try:
+            logger.debug("SSH: connecting to %s" % address)
+            client = paramiko.SSHClient()
+            client.set_missing_host_key_policy(paramiko.AutoAddPolicy())
+            client.connect(hostname=address, username=user, password=password, banner_timeout=0.5)
+            stdin, stdout, stderr = client.exec_command(command, timeout=1)
+            answer = stdout.read().decode("utf-8")
+            client.close()
+            return answer
+
+        except socket.gaierror as ex:
+            logger.error("SSH: DNS error. Could not locate %s" % address)
+            return None
+
+        except (paramiko.AuthenticationException, paramiko.PasswordRequiredException) as ex:
+            logger.error("SSH: authentication error: %s: %s" % (address, str(ex)))
+            return None
+
+        except (paramiko.BadHostKeyException,
+                paramiko.SSHException,
+                paramiko.ChannelException,
+                ) as ex:
+            logger.error("SSH: connection failed to %s: %s" % (address, str(ex)))
+            return None
+
+        except Exception as ex:
+            logger.error("SSH: unknown error at: %s. reason: %s" % (address, str(ex)))
```

Comment:
What are we doing here? No return?

Reply:
done

Reply:
Added return None like at the other exceptions...

Reply:
Catching anything strange thats not catched rightnow. If this error never is raised we are good. But didnt wanted to risk any crash because of faulty ssh connections